File: extract.js
const fs = require('fs');
const pdf = require('pdf-parse');

async function extract() {
  const ppBuffer = fs.readFileSync('./Legal/casanova-ai-privacy-policy.pdf');
  const tosBuffer = fs.readFileSync('./Legal/casanova-ai-terms-of-service.pdf');
  
  const ppData = await pdf(ppBuffer);
  fs.writeFileSync('pp.txt', ppData.text);
  
  const tosData = await pdf(tosBuffer);
  fs.writeFileSync('tos.txt', tosData.text);
}

extract();

File: extract.py
import sys
from pypdf import PdfReader

def extract_text(pdf_path, txt_path):
    reader = PdfReader(pdf_path)
    text = ""
    for page in reader.pages:
        text += page.extract_text() + "\n"
    with open(txt_path, "w") as f:
        f.write(text)

extract_text("./Legal/casanova-ai-privacy-policy.pdf", "pp.txt")
extract_text("./Legal/casanova-ai-terms-of-service.pdf", "tos.txt")
